subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/platforms.adoc

```diff
@@ -451,6 +451,10 @@ Gradle treats all entries in the `<dependencyManagement>` block of a BOM similar
 This means that any version defined in the `<dependencyManagement>` block can impact the dependency resolution result.
 In order to qualify as a BOM, a `.pom` file needs to have `<packaging>pom</packaging>` set.
 
+The BOMs declared dependencies versions are recommendations only. 
+In case you want to use another version provided by the BOM, you can set the version to the declared dependency.
+This will then override the version provided by the BOM.
```

Comment:
This will actually trigger conflict resolution, which will resolve to the highest version. So this sentence should really link to https://docs.gradle.org/current/userguide/dependency_downgrade_and_exclude.html#sec:enforcing_dependency_version and talk about conflict resolution instead of override.

I believe highlighting this more makes sense a. I would suggest making it a [NOTE] since it is already hinted at in the previous paragraph.